### PR TITLE
Develop/client env

### PR DESCRIPTION
**Pull request by kennyhyun**
Setup Client dev env

- CRA
- Typescript
- eslint; +ts
- prettier; +ts
- husky/lint-staged; autofix eslint/prettier
- Material Ui
- Storybook
- Storyshots; jss identifier, split snapshots

### CODE REVIEW

File: webapp/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    node: true,
+  },
+  plugins: ['@typescript-eslint', 'react'],
+  rules: {
+    '@typescript-eslint/no-empty-function': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    'react/prop-types': 'off', // Disable prop-types as we use TypeScript for type checking
+    '@typescript-eslint/no-unused-vars': 'warn',
+    // '@typescript-eslint/no-var-requires': 0,
+    // '@typescript-eslint/ban-ts-comment': 0,
```
Comment (taewdy) [inline webapp/.eslintrc.js]: Remove unused code please

File: webapp/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (taewdy) [inline webapp/README.md]: Remove please. We can update READEME later but Let's not have generated ones commit to the master branch.

File: webapp/package.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
+{
+  "name": "filemanager.webapp",
```
Comment (taewdy) [inline webapp/package.json]: update needed?

File: webapp/public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (taewdy) [inline webapp/public/index.html:1]: Let's remove generated page please

**Review by taewdy — APPROVED** [review]

**Comment by kennyhyun** [comment]
[DevEnv/Frontend](https://trello.com/c/fb2cxYi1/7-devenv-frontend)

File: webapp/public/index.html (hunk 4 above)
Comment (kennyhyun) [reply webapp/public/index.html:1]: I'll remove comments only since this is a necessary file

**Pull request merged**